public/scss/_app-error.scss

```diff
@@ -1,12 +1,13 @@
 .app-error {
-    background: rgba(255,255,255,0.8);
+    background: rgba(255,255,255,0.9);
     bottom: 0;
     height: 100%;
     left: 0;
+    padding: 10px;
     position: absolute;
     right: 0;
     top: 0;
-    z-index: 10;
+    z-index: 30;
```

Comment:
Yeah, the thing I need to trump was 10 so it's not 99999999 :). But yeah we could deffo consider using something fancy if it gets to be an issue. Probably just vars would do it.